examples/react-apollo/src/App.tsx

```diff
@@ -57,8 +57,22 @@ function App() {
             }
           />
           <Route path="/about" element={<AboutPage />} />
-          <Route path="/sign-in/*" element={<SignInPage />} />
-          <Route path="/sign-up/*" element={<SignUpPage />} />
+          <Route
+            path="/sign-in/*"
+            element={
+              <PublicGate>
+                <SignInPage />
+              </PublicGate>
+            }
+          />
+          <Route
+            path="/sign-up/*"
+            element={
+              <PublicGate>
+                <SignUpPage />
+              </PublicGate>
```

Comment:
Do we really need a `PublicGate` wrapper?

Reply:
Not much. It redirects authentication-related routes to index so we never show authentication routes (e.g. `/sign-in`) to an already authenticated user.
It's only in the example, but I can remove it if you want

Reply:
At first, I thought it looked strange but I tried GitHub and Vercel and they redirect users from the login page to `/` so I think we should keep it 👍